README.md

```diff
@@ -864,6 +864,6 @@ TwitterCLDR currently supports localization of certain textual objects in JavaSc
 
 ## License
 
-Copyright 2015 Twitter, Inc.
+Copyright 2014 Twitter, Inc.
```

Comment:
Hacking back in time?
![hacking](http://musictechfest.net/wp-content/uploads/2015/05/geek.jpg)